pkg/apis/hive/v1alpha1/syncset_types.go

```diff
@@ -175,6 +219,16 @@ type SyncSetCommonSpec struct {
 	// Patches is the list of patches to apply.
 	// +optional
 	Patches []SyncObjectPatch `json:"patches,omitempty"`
+
+	// SyncResources is the list of objects to sync from existing resources.
+	// +optional
+	SyncResources []SyncResource `json:"syncResources,omitempty"`
+
+	// SyncResourceApplyMode indicates if the SyncResource apply mode is "upsert" (default) or "sync".
+	// ApplyMode "upsert" indicates create and update.
+	// ApplyMode "sync" indicates create, update and delete.
+	// +optional
+	SyncResourceApplyMode SyncSetResourceApplyMode `json:"syncResourceApplyMode,omitempty"`
```

Comment:
Why does the apply mode need to be different for the ones synced from existing resources? Is there a use case for applying one way from the raw data and another way from the existing resource within the same syncset?

Reply:
I don't think we need a separate apply mode. I'll remove this one.